package.json

```diff
@@ -25,95 +25,99 @@
   "homepage": "https://github.com/Lumeer/web-ui#readme",
   "private": true,
   "dependencies": {
-    "@angular/animations": "^5.1.0",
-    "@angular/cdk": "^5.0.0",
-    "@angular/common": "^5.1.0",
-    "@angular/compiler": "^5.1.0",
-    "@angular/core": "^5.1.0",
-    "@angular/forms": "^5.1.0",
-    "@angular/http": "^5.1.0",
-    "@angular/platform-browser": "^5.1.0",
-    "@angular/platform-browser-dynamic": "^5.1.0",
-    "@angular/router": "^5.1.0",
+    "@angular/animations": "^5.1.2",
+    "@angular/cdk": "^5.0.2",
+    "@angular/common": "^5.1.2",
+    "@angular/compiler": "^5.1.2",
+    "@angular/core": "^5.1.2",
+    "@angular/forms": "^5.1.2",
+    "@angular/http": "^5.1.2",
+    "@angular/platform-browser": "^5.1.2",
+    "@angular/platform-browser-dynamic": "^5.1.2",
+    "@angular/router": "^5.1.2",
     "@ngrx/effects": "^4.1.1",
     "@ngrx/entity": "^4.1.1",
     "@ngrx/router-store": "^4.1.1",
     "@ngrx/store": "^4.1.1",
     "@ngrx/store-devtools": "^4.1.1",
-    "bootstrap": "4.0.0-beta.2",
-    "core-js": "^2.4.1",
     "minigrid": "^3.1.1",
     "ng-click-outside": "^3.1.1",
-    "ng-snotify": "^4.0.2",
-    "ngx-contextmenu": "^3.0.0",
+    "ng-snotify": "^4.2.0",
+    "ngx-contextmenu": "^4.1.1",
     "ngx-quill": "^2.0.4",
     "parchment": "^1.1.3",
-    "popper.js": "1.12.3",
     "quill": "^1.3.4",
-    "quill-delta-to-html": "^0.5.1",
+    "quill-delta-to-html": "^0.5.5",
+    "angular2-notifications": "^0.9.7",
+    "bootstrap": "4.0.0-beta.2",
+    "core-js": "^2.5.3",
+    "lodash": "^4.17.4",
+    "ngx-bootstrap": "^2.0.0-rc.0",
```

Comment:
Some dependencies such as `angular2-notifications` or `ngx-bootstrap` have already been removed from this project.